src/generic_core/uproxy_core.ts

```diff
@@ -655,15 +655,64 @@ export class uProxyCore implements uproxy_core_api.CoreApi {
       };
       const MAX_INSTALLS = 5;
       return retry(install, MAX_INSTALLS);
-    }).then((output: string) => {
+    }).then((cloudNetworkData :any) => {
```

Comment:
Right. I think this is a good argument for exporting it though - accessing the fields via the `[]` operator is a code smell. It's a one liner and I can jam it into the upcoming release.